fedn/fedn/utils/plugins/androidhelper.py

```diff
@@ -0,0 +1,108 @@
+import json
+import os
+import tempfile
+
+import numpy as np
+
+from .helperbase import HelperBase
+
+
+class Helper(HelperBase):
+    """FEDn helper class for android json model weights."""
+
+    def __init__(self):
+        """Initialize helper."""
+        self.name = "androidhelper"
+        super().__init__()
+
+    # function to calculate an incremental weighted average of the weights
+    def increment_average(
```

Comment:
Why do we need two version of this function? The implementation in increment_average_add looks more efficient (vectorized). Why not have only that one (as 'increment_average')?

Reply:
Think this is very much still in dev state. Will convert to draft.